packages/main/package.json

```diff
@@ -22,6 +22,8 @@
   "dependencies": {
     "@ui5/webcomponents": "1.0.0-rc.2",
     "@ui5/webcomponents-react-base": "0.3.2-rc.14",
+    "lodash.debounce": "^4.0.8",
+    "lodash.throttle": "^4.1.1",
```

Comment:
not needed anymore

Reply:
`throttle` is removed, `debounce` is used